crates/codegen/spec/src/html_ebnf_writer.rs

```diff
@@ -40,12 +40,15 @@ impl<T: Write> EBNFWriter for HTMLEBNFWriter<'_, T> {
         .unwrap();
     }
 
-    fn write_production_definition(&mut self, production_name: &str) {
+    fn write_production_definition(&mut self, metadata: Option<&str>, production_name: &str) {
```

Comment:
In my next PR, I'm actually modifying this bit, as the spec linking was broken in #332.
We need heirarchial information here:

- Block or more...
  - Version or more...
    - Statement or more...
      - Token or more...

It is a bigger change to the ebnf writer, and solving the root issue of heirarchy.
If you don't mind, I suggest withholding that bit for now, until the bigger fix is ready for review.

Reply:
ok